base/pkg/query.jl

```diff
@@ -66,6 +66,73 @@ function dependencies(avail::Dict, fix::Dict = Dict{String,Fixed}("julia"=>Fixed
     avail, conflicts
 end
 
+function partial_update_mask(instd::Dict{String,Tuple{VersionNumber,Bool}}, avail::Dict{String,Dict{VersionNumber,Available}}, upkgs::Set{String})
```

Comment:
It's the `fixed` tag as returned by `Read.installed`, see [here](https://github.com/JuliaLang/julia/blob/38c803d2252736612878ccf5b040fb35c4bfa516/base/pkg/read.jl#L201-L217).

I tend to spell out the type signatures in full in these functions (rather than just writing `instd::Dict`) mostly for "documentation" purposes.